### ISSUE TITLE
Remove redundant aria-disabled attributes

### ISSUE THREAD

**Issue by AresEkb** [eclipse-sirius/sirius-components#6532]
* [x] **I have checked that this bug has not yet been reported by someone else**
* [x] **I have checked that this bug appears on Chrome**
* [x] **I have specified the version** :
- latest
* [x] **I have specified my environment** : 
- my computer: Linux

### Screenshots

No

### Steps to reproduce

Inspect attributes of some HTML elements.

### Expected behavior

Enabled menu items doesn't have aria-disabled attributes. And they are clickable by Playwright.

### Actual behavior

Some elements have redundant aria-disabled attributes. Some elements have conflicting disabled="false" and aria-disabled="true".
